tests.lisp

```diff
@@ -5,6 +5,46 @@
 
 (in-package :split-sequence/tests)
 
+#+sbcl
+(progn
+  (eval-when (:compile-toplevel :load-toplevel :execute)
```

Comment:
Is the EVAL-WHEN necessary ?

Reply:
It was necessary.  I think this is a bug in SBCL, but the class is not available in the following method definitions without that, and warnings are issued.